ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -62,80 +49,93 @@ const FAM_UPLOAD_FORMATS = [].concat(BASE_UPLOAD_FORMATS)
+          </Table.Cell>
+        </Table.Row>
+        <Table.Row><Table.Cell /></Table.Row>
+        {uploadFormats.map(({ title, ext, formatLinks }) =>
+          <Table.Row key={title}>
+            <TableHeaderCell>
+              {title} ({formatLinks ? formatLinks.map(
+                ({ href, linkExt }, i) => <span key={linkExt}>{i > 0 && ' / '}<a href={href} target="_blank">.{linkExt}</a></span>)
+                : `.${ext}`})
+            </TableHeaderCell>
+            <Table.Cell>
+              {ext &&
+                <span>
+                  download &nbsp;
+                  {blankExportConfig &&
                   <span>
-                    download &nbsp;
-                    {blankExportConfig &&
-                    <span>
-                      template: <FileLink data={blankExportConfig} ext={ext} linkContent="blank" /> or&nbsp;
-                    </span>
-                    }
-                    <FileLink data={exportConfig} ext={ext} linkContent="current individuals" />
+                    template: <FileLink data={blankExportConfig} ext={ext} linkContent="blank" /> &nbsp;
                   </span>
-                }
-              </TableCell>
-            </TableRow>,
-          )}
-        </Table.Body>
-      </StyledTable>
-
-      The table must have a header row with the following column names.<br />
-      <br />
-      <div>
-        <BoldText>Required Columns:</BoldText><br />
-        <StyledTable>
-          <Table.Body>
-            {requiredFields.map(field =>
-              <TableRow key={field.header}>
-                <TableCell><BoldText>{field.header}</BoldText></TableCell>
-                <TableCell />
-              </TableRow>,
-            )}
-          </Table.Body>
-        </StyledTable>
-        <BoldText>Optional Columns:</BoldText>
-        <StyledTable>
-          <Table.Body>
-            {optionalFields.map(field =>
-              <TableRow key={field.header}>
-                <TableCell><BoldText>{field.header}</BoldText></TableCell>
-                <TableCell>{field.description}</TableCell>
-              </TableRow>,
-            )}
-          </Table.Body>
-        </StyledTable>
-      </div>
-      {details && <div><br />{details}</div>}
-      <br />
-    </Container>
-    <br />
+                  }
+                  {exportConfig &&
+                  <span>
+                    or <FileLink data={exportConfig} ext={ext} linkContent="current individuals" />
+                  </span>
+                  }
+                </span>
+              }
+            </Table.Cell>
+          </Table.Row>,
+        )}
+        <Table.Row><Table.Cell /></Table.Row>
+        <Table.Row>
+          <Table.Cell colSpan={2}>
+            The table must have a header row with the following column names.
+          </Table.Cell>
+        </Table.Row>
+        <Table.Row>
+          <Table.Cell colSpan={2}>
+            <BoldText>Required Columns:</BoldText>
```

Comment:
You can swap out all the `<Table.Cell><BoldText>...</BoldText></Table.Cell>` for  `<Table.HeaderCell>...</Table.HeaderCell>`